public/js/components/AtomEmbed/CreateTargetForm.js

```diff
@@ -67,9 +67,6 @@ class CreateTargetForm extends React.Component {
           <ManagedField fieldLocation="title" name="Title" isRequired={true}>
             <FormFieldTextInput/>
           </ManagedField>
-          <ManagedField fieldLocation="description" name="Description" isRequired={true}>
```

Comment:
Just had a look at the targeting tool and it looks like this is optional, which is good. @currysoup , am I correct?